src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -186,6 +187,17 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('acr task create') as c:
         c.argument('task_name', completer=None)
 
+    for scope in ['acr task credential add', 'acr task credential update']:
+        with self.argument_context(scope) as c:
+            # Custom registry credentials
+            c.argument('login_server', help="The login server of the custom registry. For instance, 'myregistry.azurecr.io'.", required=True)
+            c.argument('username', options_list=['--username', '-u'], help='The username to login to the custom registry.', required=True)
+            c.argument('password', options_list=['--password', '-p'], help='The password to login to the custom registry.', required=True)
+
+    with self.argument_context('acr task credential remove') as c:
+        # Custom registry credentials
+        c.argument('login_server', help="The login server of the custom registry. For instance, 'myregistry.azurecr.io'.", required=True)
```

Comment:
Can the three new params be grouped under `acr task credential` so `login_server` doesn't need to be duplicated?

Reply:
`acr task credential list` does not need --login-server

Reply:
It doesn't show unless the command method has the parameter.